app/views/event_instances/_hangout_button.html.erb

```diff
@@ -8,8 +9,32 @@
     'callbackUrl' => hangout_url('id').gsub(/id$/, '')
   }) %>
```

Comment:
Moved all changes from this partial to view. So this file is untouched by me right now.